plugins/techdocs-backend/src/service/router.ts

```diff
@@ -226,13 +231,35 @@ export async function createRouter(
     );
   });
 
+  // Route treated similarly to /static/docs
+  // Requires an existing entity in the catalog for authorization
+  router.use(
```

Comment:
Hmm...  Since `publisher.docsRouter()` implementations basically just expose the entire underlying storage, if we allow `/static/docs` and `/static/entity/docs` to be run in parallel, wouldn't that open up the possibility for TechDocs content to leak by someone just removing `/entity` from the URL?

Reply:
Yup you're right. Seems like we need to enforce this for all `/static/docs` requests and require a different approach for non-catalog docs (eg by extending the techdocs router). Given most of the techdocs functionality already relies on entity refs, perhaps this is okay?